util/pace/util/_properties.py

```diff
@@ -16,437 +16,614 @@
     },
     "x_wind": {
         "dims": [Z_DIM, Y_INTERFACE_DIM, X_DIM],
         "restart_name": "u",
+        "driver_name": "u",
         "units": "m/s",
+        "restart_file": "fv_core.res.tile",
     },
     "x_wind_on_c_grid": {
         "dims": [Z_DIM, Y_DIM, X_INTERFACE_DIM],
         "restart_name": "uc",
+        "driver_name": "uc",
         "units": "m/s",
     },
     "y_wind": {
         "dims": [Z_DIM, Y_DIM, X_INTERFACE_DIM],
         "restart_name": "v",
+        "driver_name": "v",
         "units": "m/s",
+        "restart_file": "fv_core.res.tile",
     },
     "y_wind_on_c_grid": {
         "dims": [Z_DIM, Y_INTERFACE_DIM, X_DIM],
         "restart_name": "vc",
+        "driver_name": "vc",
         "units": "m/s",
     },
+    "specific_humidity": {
+        "dims": [Z_DIM, Y_DIM, X_DIM],
+        "restart_name": "sphum",
+        "driver_name": "qvapor",
+        "units": "kg/kg",
+        "restart_file": "fv_tracer.res.tile",
+    },
+    "cloud_water_mixing_ratio": {
+        "dims": [Z_DIM, Y_DIM, X_DIM],
+        "restart_name": "liq_wat",
+        "driver_name": "qliquid",
+        "units": "kg/kg",
+        "restart_file": "fv_tracer.res.tile",
+    },
+    "cloud_ice_mixing_ratio": {
+        "dims": [Z_DIM, Y_DIM, X_DIM],
+        "restart_name": "ice_wat",
+        "driver_name": "qice",
+        "units": "kg/kg",
+        "restart_file": "fv_tracer.res.tile",
+    },
+    "rain_mixing_ratio": {
+        "dims": [Z_DIM, Y_DIM, X_DIM],
+        "restart_name": "rainwat",
+        "driver_name": "qrain",
+        "units": "kg/kg",
+        "restart_file": "fv_tracer.res.tile",
+    },
+    "snow_mixing_ratio": {
+        "dims": [Z_DIM, Y_DIM, X_DIM],
+        "restart_name": "snowwat",
+        "driver_name": "qsnow",
+        "units": "kg/kg",
+        "restart_file": "fv_tracer.res.tile",
+    },
+    "graupel_mixing_ratio": {
+        "dims": [Z_DIM, Y_DIM, X_DIM],
+        "restart_name": "graupel",
+        "driver_name": "qgraupel",
+        "units": "kg/kg",
+        "restart_file": "fv_tracer.res.tile",
+    },
+    "ozone_mixing_ratio": {
+        "dims": [Z_DIM, Y_DIM, X_DIM],
+        "restart_name": "o3mr",
+        "driver_name": "qo3mr",
+        "units": "kg/kg",
+        "restart_file": "fv_tracer.res.tile",
+    },
+    "turbulent_kinetic_energy": {
+        "dims": [Z_DIM, Y_DIM, X_DIM],
+        "restart_name": "sgs_tke",
+        "driver_name": "qsgs_tke",
+        "units": "kg/kg",
+        "restart_file": "fv_tracer.res.tile",
+    },
+    "cloud_fraction": {
+        "dims": [Z_DIM, Y_DIM, X_DIM],
+        "restart_name": "cld_amt",
+        "driver_name": "qcld",
+        "units": "kg/kg",
+        "restart_file": "fv_tracer.res.tile",
+    },
+    "vertical_thickness_of_atmospheric_layer": {
+        "dims": [Z_DIM, Y_DIM, X_DIM],
+        "restart_name": "DZ",
+        "driver_name": "delz",
+        "units": "m",
+        "restart_file": "fv_core.res.tile",
+    },
```

Comment:
In the fortran restart loading code, these are treated as "tracer properties", since they're defined at runtime by the tracer configuration file. These aren't statically-defined.

Reply:
Because we hard-code the tracer configuration in everything we do currently in Pace, you can define a static/constant TRACER_PROPERTIES dictionary somewhere inside `pace.driver` (possibly in the restart initialization code) which we could later generate from the microphysics configuration.